domains/transfer/TransferAgreementAutomation/Worker.UnitTests/Service/ProjectOriginWalletServiceTest.cs

```diff
@@ -138,18 +153,23 @@ public async Task
         TransferCertificates_CertificateEndDateAfterTAStartDatCertificateStartDateIsBefore_ShouldNotCallWalletTransferCertificate()
     {
         var now = DateTimeOffset.UtcNow;
-        var transferAgreement = new TransferAgreementDto(
-            EndDate: DateTimeOffset.UtcNow.AddHours(3).ToUnixTimeSeconds(),
-            ReceiverReference: Guid.NewGuid().ToString(),
-            ReceiverTin: "12345678",
-            SenderId: Guid.NewGuid().ToString(),
-            StartDate: now.ToUnixTimeSeconds()
-        );
+        var transferAgreement = new TransferAgreement
+        {
+            EndDate = DateTimeOffset.UtcNow.AddHours(3),
+            ReceiverReference = Guid.NewGuid(),
+            ReceiverTin = "12345678",
+            SenderId = Guid.NewGuid(),
+            StartDate = now,
+            Id = Guid.NewGuid(),
+            SenderName = "SomeSender",
+            SenderTin = "11223344",
+            TransferAgreementNumber = 0
+        };
 
```

Comment:
You are using the exact same test data for many of the tests. Can't you create a separate helper method, for it? Seems cluttered to have it written in each test.

Reply:
corrected